Core/Core/Logging/SpeckleLog.cs

```diff
@@ -171,9 +166,19 @@ SpeckleLogConfiguration logConfiguration
     var canLogToFile = true;
     s_logFolderPath = SpecklePathProvider.LogFolderPath(hostApplicationName, hostApplicationVersion);
     var logFilePath = Path.Combine(s_logFolderPath, "SpeckleCoreLog.txt");
+    var id = GetUserIdFromDefaultAccount();
+    var fileVersionInfo = GetFileVersionInfo();
     var serilogLogConfiguration = new LoggerConfiguration().MinimumLevel
       .Is(logConfiguration.MinimumLevel)
       .Enrich.FromLogContext()
+      .Enrich.WithProperty("id", id)
+      .Enrich.WithProperty("version", fileVersionInfo.FileVersion)
+      .Enrich.WithProperty("productVersion", fileVersionInfo.ProductVersion)
+      .Enrich.WithProperty("hostOs", DetermineHostOsSlug())
+      .Enrich.WithProperty("hostOsVersion", Environment.OSVersion)
+      .Enrich.WithProperty("hostOsArchitecture", RuntimeInformation.ProcessArchitecture.ToString())
+      .Enrich.WithProperty("runtime", RuntimeInformation.FrameworkDescription)
+      .Enrich.WithProperty("hostApplication", $"{hostApplicationName}{hostApplicationVersion ?? ""}")
       .Enrich.FromGlobalLogContext();
```

Comment:
If we enrich from the global log context, and this context has been polluted by a another plugin, does this mean we'd be getting properties that are not ours?

We should prevent this as much as preventing our properties leaking to other Sentry consumers.

Reply:
That's a valid point, I'll test it without this enrichment.